### PR TITLE
📝 Write initial blog post announcing speaker mentors

### PR DESCRIPTION
**Pull request by drewbrew**
Still waiting on a bio from Josue and Mason

Changes proposed in this PR:

- Writes a blog post introducing our 5 speaker mentors

### CODE REVIEW

**Review by jefftriplett — APPROVED** [review]
🎉  LGTM

File: _posts/2022-05-16-introducing-speaker-mentors.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+image: /static/img/blog/cfp-announced-2022.png
+post_photo_url: /static/img/blog/speaking-2022-cfp.jpg
+post_photo_alt: "Speaker addressing a crowd at DjangoCon US 2019 in San Diego"
+title: "Introducing our speaker mentors for DjangoCon US 2022!"
+---
+
+The DjangoCon 2022 organizers are excited to announce the speaker mentors for the 2022 [call for proposals](https://pretalx.com/djangocon-2022/cfp)!
+
+## Mentors
+- [Josue Balandrano Coronel](mailto:josuebc@defna.org), (HEY YOU DO NOT PUBLISH UNTIL HE SENDS AN UPDATED BIO)
```
Comment (drewbrew) [inline _posts/2022-05-16-introducing-speaker-mentors.md]: @xirdneh reminder

Comment (logankilpatrick) [reply _posts/2022-05-16-introducing-speaker-mentors.md]: Just pinged Josue on Slack again

Comment (jefftriplett) [inline _posts/2022-05-16-introducing-speaker-mentors.md]:
```suggestion
- [Josue Balandrano Coronel](mailto:josuebc@defna.org), is a DEFNA director, DjangoCon US organizer, and a software engineer at the Texas Advanced Computing Center.

**Review by jefftriplett — APPROVED** [review]

**Pull request merged**